lib/viewcomponents/node.jsx

```diff
@@ -64,20 +66,20 @@ var Node = React.createClass({
   },
 
   _onNodeBodyPseudoDrag: function(event) {
-      var oldPos = this.props.model.getPosition();
+      var oldPos = this.props.model.get('position');
       
       var newX = oldPos.x + event.movementX;
       var newY = oldPos.y + event.movementY;
 
       // Change the position of the node in the model.
-      this.props.globalModel.moveNode(this.props.model.getId(), newX, newY);
+      this.props.model.setPosition(newX, newY);
```

Comment:
I see. It made me wonder if I understood the setter syntax. I do think that consistency would be better, so no convenience method. Someone picking this thing up and trying to work on it might have the same confusion that I had, without the benefit of having been around to see this PR and discuss it.